.github/julia/build_tarballs_release.jl

```diff
@@ -136,6 +137,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    Dependency(PackageSpec(name="HiGHS_jll", uuid="8fd58aa0-07eb-5a78-9b36-339c94fd15ea")),
```

Comment:
```suggestion
    Dependency(PackageSpec(name="HiGHS_jll", uuid="8fd58aa0-07eb-5a78-9b36-339c94fd15ea"), compat="1"),
```

Reply:
@odow It's not needed in that case because we don't use it in the Julia ecosystem, so the shared libraries can't be automatically upgraded and replaced.

If the CI build with Julia tests passed, it's safe to generate binaries for a use outside Julia when a new release is tagged.